db/queries/reporting/queries.py

```diff
@@ -58,6 +58,22 @@ def export_json_to_csv(return_data, headers=None):
     return bytes_output
 
 
+def export_json_to_excel(return_data: dict):
+    output = io.BytesIO()
+
+    if not return_data:
+        return output
+
+    with pd.ExcelWriter(output, engine="openpyxl") as writer:
+        for key in return_data.keys():
+            df = pd.DataFrame(return_data[key])
+            df.to_excel(writer, sheet_name=key)
+
+    # seeking was necessary. Python 3.5.2, Flask 0.12.2
```

Comment:
Do we need this? We aren't in python 3.5.2

Reply:
Also should the api layer be returning stuff formatted in excel? That feels like more of a frontend/consumer thing, and the backend should just supply json....

Reply:
seeking is necessary from python 3.5.2 so weed need that line of code.

Intially I went with json output from API. But this caused dict keys to scramble and messed up excel fields structure in assessment. There is `OrderedDict` module which can solve this but complicates the implementation. so for now I would to keep it simple & return the excel file from application-store API